content/learning-paths/smartphones-and-mobile/using_unity_machine_learning_agents/08-Training-our-ML-agents.md

```diff
@@ -0,0 +1,540 @@
+    keep_checkpoints: 30
+    checkpoint_interval: 100000
+    max_steps: 3000000
+    time_horizon: 64
+    summary_freq: 5000
+    self_play:
+      save_steps: 20000
+      team_change: 100000
+      swap_steps: 10000
+      play_against_latest_model_ratio: 0.5
+      window: 10
+
+  Vampire:
+    trainer_type: ppo
+    hyperparameters:
+      # common to PPO and SAC
+      batch_size: 1024
+      buffer_size: 10240
+      learning_rate: 3.0e-4
+      learning_rate_schedule: linear
+      # PPO-specific
+      beta: 1.0e-2
+      epsilon: 0.2
+      lambd: 0.95
+      num_epoch: 3
+    network_settings:
+      normalize: false
+      hidden_units: 128
+      num_layers: 2
+    reward_signals:
+      extrinsic:
+        gamma: 0.99
+        strength: 1.0
+    keep_checkpoints: 30
+    checkpoint_interval: 100000
+    max_steps: 3000000
+    time_horizon: 64
+    summary_freq: 5000
+    self_play:
+      save_steps: 20000
+      team_change: 100000
+      swap_steps: 10000
+      play_against_latest_model_ratio: 0.5
+      window: 10
+```
+
+Within this yaml we have the ability to set-up 2 separate brains. One named _Paladin_ (for the knight) and another named _Vampire_ (for the NPC). Currently they have identical properties, but you could customize each to have slightly different behaviors.
+
+## Use pre-made training data
+
+We have already prepared 3 data files; one for each difficulty level. Let's plug them in:
+
+1. In the Project tab (normally at the bottom left), if not open already, open the _Level_DevSummit2022_ scene by double-clicking it:
+
+    `Assets->#DevSummit2022->Scenes->Level_DevSummit2022`
+
+1. Now select the _AgentsSettings_ object, in the _Hierachy_ tab (normally on the left).
+
+    ![Agent Settings](images/heirachy-agent-settings.png "Figure 1. Agents Settings")
+
+1. Now in the _Inspector_ (normally on the right) scroll to the _Battle Env Controller (Script)_ and you should see properties for _Easy Battle Brain_, _Normal Battle Brain_ and _Hard Battle Brain_.
+
+1. Next to the _Easy Battle Brain_ there is a circle. Click it to pop-up the _Select NN Model_ menu, as per Figure 2. 
+
+    !["NN Model Pop-up"](images/battle-enviroment-controller-battle-brain.png "Figure 2. Select an NN Model asset for each difficulty")
+    Note that you can use the slider in the pop-up to change the icon size. Drag fully to the left to reduce to list detail view if needed.
+
+1. For each brain there is already a pre-trained version you can use, or if your training has completed use that. Match up each Battle Brain property to an appropriate NN Model. We have used the name "BossBattle" in this learning path.
+
+    ![Battle Brains](images/boss-battle-brains.png "Figure 3. Brain assets set for easy, medium and hard.")
+
+1. Once the brains are set we can launch the game in the Unity Editor, to make sure everything is working and wired up correctly. If you select "Demo" mode you can see the two ML Agents battle it out.
+
```

Comment:
Suggest adding a line or two here saying something like:
"The next steps are to deploy to an Android device, and then profile for performance. But these are/will be covered in other Learning paths."
(And then add links to those Learning Paths once they are ready)

Reply:
Alternatively could have that as a "Next Steps" section on the Summary page, between "What we covered" and "Things to try"